docs/providers/aws/events/apigateway.md

```diff
@@ -1741,7 +1741,7 @@ provider:
     restApi: true
 ```
 
-The log streams will be generated in a dedicated log group which follows the naming schema `/aws/api-gateway/{service}-{stage}`.
+The log streams will be generated in a dedicated log group which follows the naming schema `/aws/api-gateway/{service}-{stage}`. Note the property `restApi` can only be true if the API Gateway service has been created inside the stack.
```

Comment:
Let's go with more descriptive documentation as the current proposal is vague. Let's start on a new line and go with something like:

```
**Note:** If external API Gateway resource is used and imported via `provider.apiGateway.restApiId` setting, `provider.logs.restApi` setting will be ignored.
```